### PR TITLE
Use yaml output on .yml files too

### LINKED ISSUE #164: Doesn't set `--output-type yaml` on `.yml` files, but only `.yaml` files
##### SUMMARY

The automatic `--output-type` is only set to `yaml` if the file extension is `.yaml`. This results in `.yml` files getting JSON formatting.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

`sops_encrypt` module

##### ANSIBLE VERSION

```paste below
ansible [core 2.15.3]
  config file = /redacted/ansible.cfg
  configured module search path = ['/home/redacted/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /redacted/.venv/lib/python3.11/site-packages/ansible
  ansible collection location = /home/redacted/.ansible/collections:/usr/share/ansible/collections
  executable location = /redacted/.venv/bin/ansible
  python version = 3.11.4 (main, Jun  6 2023, 22:16:46) [GCC 12.3.0] (/redacted/.venv/bin/python)
  jinja version = 3.1.2
  libyaml = True

```

##### COLLECTION VERSION

```paste below
Collection     Version
-------------- -------
community.sops 1.6.4
```

##### CONFIGURATION

```properties
CONFIG_FILE() = /redacted/ansible.cfg
DEFAULT_FORKS(/redacted/ansible.cfg) = 200
DEFAULT_GATHERING(/redacted/ansible.cfg) = smart
DEFAULT_JINJA2_EXTENSIONS(/redacted/ansible.cfg) = jinja2.ext.do,jinja2.ext.i18n
DEFAULT_MANAGED_STR(/redacted/ansible.cfg) = "This file is Managed by Ansible - Please do not edit by hand changes will be overwritten!"
DEFAULT_ROLES_PATH(/redacted/ansible.cfg) = ['/redacted/roles.galaxy', '/redacted/roles']
DEFAULT_TIMEOUT(/redacted/ansible.cfg) = 30
DEFAULT_VAULT_IDENTITY_LIST(env: ANSIBLE_VAULT_IDENTITY_LIST) = ['/redacted/scripts/ansible-vault-keyring-client.py']
EDITOR(env: EDITOR) = nvim
HOST_KEY_CHECKING(/redacted/ansible.cfg) = False
INTERPRETER_PYTHON(/redacted/ansible.cfg) = /usr/bin/env python3
PAGER(env: PAGER) = less --raw-control-chars --quit-if-one-screen
VARIABLE_PLUGINS_ENABLED(/redacted/ansible.cfg) = ['host_group_vars', 'community.sops.sops']

```

##### OS / ENVIRONMENT

Linux 6.1.56
x86_64

##### STEPS TO REPRODUCE

Run the following playbook:

```yaml
- hosts: localhost
  gather_facts: false
  tasks:
    - name: Create key vault file
      community.sops.sops_encrypt:
        path: "foo.yml"
        content_yaml:
          foo: bar
```

##### EXPECTED RESULTS

Content of `foo.yml`:
```yaml
foo: ENC[AES256_GCM,data:rElC,iv:vBbFgYuKvnCrEjX2plYUvtcYmFtBpl3UBKa8xJfEv/c=,tag:oijrsyuNsykJ8J8KFVXMlQ==,type:str]
sops:
    kms: []
    gcp_kms: []
    azure_kv: []
    hc_vault: []
...snip
```

##### ACTUAL RESULTS

Content of `foo.yml`:
```json
{
	"foo": "ENC[AES256_GCM,data:g0fG,iv:Bi+6De1oeqJEUnDN0zNa9me5Q0S07iOcN30eP9Z95YE=,tag:myrbR+PDl23dX0vmbP5+5A==,type:str]",
	"sops": {
		"kms": null,
		"gcp_kms": null,
		"azure_kv": null,
...snip
```

### PR DESCRIPTION
**Pull request by applejag**
### Motivation

Closes #164

### Changes description

- Changed `sops_encrypt` module to also use `yaml` output on `.yml` files

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by applejag** [comment]
> Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

Added

File: changelogs/fragments/165-yaml-output-for-yml-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (felixfontein) [inline changelogs/fragments/165-yaml-output-for-yml-extension.yaml:1]: Looks like yamllint doesn't like this empty line :)
```suggestion
```

File: changelogs/fragments/165-yaml-output-for-yml-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+bugfixes:
+  - sops_encrypt - fix output-type not set to ``yaml`` when file ended with ``.yml``. Now both ``.yaml`` and ``.yml`` files use the SOPS ``--output-type=yaml`` formatting. (https://github.com/ansible-collections/community.sops/issues/164).
```
Comment (felixfontein) [inline changelogs/fragments/165-yaml-output-for-yml-extension.yaml:3]:
```suggestion
  - sops_encrypt - ensure that output-type is set to ``yaml`` when the file extension ``.yml`` is used. Now both ``.yaml`` and ``.yml`` files use the SOPS ``--output-type=yaml`` formatting (https://github.com/ansible-collections/community.sops/issues/164).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@applejag thanks a lot for your contribution!

**Pull request merged**